src/sequelize.js

```diff
@@ -900,7 +907,7 @@ class Sequelize {
       ...options,
     };
 
-    await this.query('SELECT 1+1 AS result', options);
+    await this.query(`SELECT 1+1 AS result${this.options.dialect === 'ibmi' ? ' FROM SYSIBM.SYSDUMMY1' : ''}`, options);
```

Comment:
can we add a method on queryInterface (queryGenerator?) to generate this query from the code that's controlled by the dialect instead?